Optimized description

EricBergDE opened the pull request:
Changed description to fit to the actual sign in procedure.
Clarified on usage of Public Keys, because customers are asking where keys are from.

azurecla commented:
Hi __@4c74356b41__, I'm your friendly neighborhood Azure Pull Request Bot (You can call me AZPRBOT). Thanks for your contribution!
    <span>
        This seems like a small (but important) contribution, so no contribution license agreement is required at this point. Real humans will now evaluate your PR.
    </span>

TTYL, AZPRBOT;

PRmerger commented:
@4c74356b41 : Thanks for your contribution to the Azure documentation! The author, @mahesh-unnikrishnan, has been notified to review your proposed change.

mahesh-unnikrishnan commented:
Thanks for bringing this to our notice. I've resolved this typo as part of a recent doc update.

4c74356b41 commented:
should I close this?

mahesh-unnikrishnan commented:
Please go ahead and close this PR.

EricBergDE committed:
Optimized description

Changed description to fit to the actual sign in procedure.
Clarified on usage of Public Keys, because customers are asking where keys are from.
